nemo_skills/evaluation/metrics.py

```diff
@@ -160,6 +162,17 @@ def update(self, predictions, aggregation_mode):
             if self.has_judge:
                 current_correct_judge += is_correct_judgement(predictions[0]['judgement'])
             self.no_answer += predictions[0]['predicted_answer'] is None
+        elif aggregation_mode == "reward":
```

Comment:
we can also add another aggregation, called majority-reward or something like that, where you would re-weight the answers for rm scores and still take majority. So same as in regular majority, but each answer contributes more or less depending on it's scores (which we should normalize to sum to 1)